.github/workflows/ubuntu_18_04.yml

```diff
@@ -0,0 +1,73 @@
+name: Kinetic & Melodic ROS CI
+
+on: [push]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os_distro: [[ubuntu-16.04, kinetic], [ubuntu-18.04, melodic]]
+    runs-on: ${{ matrix.os_distro[0] }}
+    env:
+      ROS_CI_DESKTOP: "`lsb_release -cs`"  # e.g. [trusty|xenial|...]
+      ROS_DISTRO: ${{ matrix.os_distro[1] }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: catkin_ws/src/ros1_lifecycle
+      ### ROS ###
+      - name: Setup for ROS ${{ matrix.os_distro[1] }} install
```

Comment:
This should be removed when using the library/ros base containers suggested above.